docs/checkers/index.md

```diff
@@ -2557,6 +2557,18 @@ and `pdb.set_trace()`) are found. These breakpoints should be removed in product
 
 ```
 
+## Modified Iteration
+
+(E4702)=
+
+### Modified iterating dict
```

Comment:
Please add the error code to the heading itself (see other error headings on this page as examples for format to follow)